_posts/merging_strategy.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Merging strategy
+author: antoine-wdg-rmz
+tags: [git]
+---
+## Merging strategy
+
+When wanting to merge branch `feature-x`:
+
+```bash
+git checkout feature-x
+git fetch origin
+git rebase origin/develop [-i]
+git push origin feature-x [-f]
+```
+
+Then open a PR on Bitbucket, and merge and **force a merge commit** (when merging manually,
```

Comment:
La phrase peut sans doute être remplacée par une liste de points ? (2 "and" s'enchaînent).